document/js-api/index.bs

```diff
@@ -384,53 +398,104 @@ Note:
 </div>
 
 <div algorithm>
-  The <dfn method for="WebAssembly">validate(|bytes|)</dfn> method, when invoked, performs the following steps:
+  To <dfn>validate builtins and imported string for a WebAssembly module</dfn> from module |module|, enabled builtins |builtinSetNames|, and |importedStringModule|, perform the following steps:
```

Comment:
For new algorithms, please follow the formatting of "retrieving a host value", that is, empty line after `<div>` and before `</div>`, an empty line between the header and the list, and start on column 0. (I haven't made the spec consistent yet because of diff churn, but I probably should after this batch of proposals.)